amethyst_config/src/lib.rs

```diff
@@ -1,20 +1,19 @@
 //! Loads RON files into a structure for easy / statically typed usage.
-//!
 
 #![crate_name = "amethyst_config"]
 #![doc(html_logo_url = "https://www.amethyst.rs/assets/amethyst.svg")]
 #![warn(missing_docs, rust_2018_idioms, rust_2018_compatibility)]
 
 #[macro_use]
 extern crate log;
-use ron;
 
 use std::{
     error::Error,
     fmt, io,
     path::{Path, PathBuf},
 };
 
+use ron;
```

Comment:
I thought we could drop these now with rust 2018?

Reply:
2018 allows us to remove `extern crate`s when the crate in question is 2018 or doesn't need `#[macro_use]`. 2015 crates that export macros still require `extern crate`s